libs/components/inline-form/src/lib/modules/inline-form/types/inline-form-close-args.ts

```diff
@@ -1,7 +1,9 @@
 export interface SkyInlineFormCloseArgs {
   /**
-   * Returns a `string` value that explains why the inline form closed.
-   * The possible values are `cancel`, `delete`, `done`, and `save`.
+   * Returns a `string` value to explain why users clicked a custom button and initiated a `close` event.
+   * This correlates to the `action` in `SkyInlineFormButtonConfig`. The possible values are `cancel`, `delete`, `done`, `save`,
+   * and any custom values provided through `action`. The close event is automatic for the standard action values, but
```

Comment:
The `reason` is automatic for the standard actions, but the actual action taken from that is not automatic - you still have to right code to act upon the `reason`. The current sentence here doesn't read like that to me. If I was a consumer, I'm worried they will think that they do not need to write code for those standard actions.

Reply:
oh good catch, maybe an update thats something along the lines of `consumers must hook into the event for all actions to be taken on the close event firing` @blackbaud-johnly thoughts?

Reply:
@Blackbaud-TrevorBurch, does that address your concern? I think so, but I wonder if it would be clearer if we fleshed it out a little further. What do y'all think of: `consumers must hook into the `close` event for custom values and must write code for all buttons to act on the `reason` when the `close` event fires.`